pdns/dnsmessage.proto

```diff
@@ -126,6 +126,36 @@ message PBDNSMessage {
     required MetaValue value = 2;
   }
   repeated Meta meta = 22;                      // Arbitrary meta-data - to be used in future rather than adding new fields all the time
+
+  // The well known EventTrace event numbers
+  enum EventType {
+    CustomEvent = 0;                            // A custom event
+    RecRecv = 1;                                // A request was received by the recursor process
+    PCacheCheck = 2;                            // A packet cache check was initiated or completed; value: bool cacheHit
+    SyncRes = 3;                                // Syncres main function has started or completed; value: int rcode
```

Comment:
I wonder if it would make sense to move events specific to the recursor to the 2xx range?

Reply:
Yes. Will do so.